src/AzureExtension/DeveloperId/AuthenticationHelper.cs

```diff
@@ -34,20 +34,27 @@ public AuthenticationSettings MicrosoftEntraIdSettings
 
     public static Guid TransferTenetId { get; } = new("f8cdef31-a31e-4b4a-93e4-5f571e91255a");
 
-    [System.Diagnostics.CodeAnalysis.SuppressMessage("Globalization", "CA1305:Specify IFormatProvider", Justification = "Globalization not required as the strings are not being displayed to user and are used in initialization")]
     public AuthenticationHelper()
     {
         MicrosoftEntraIdSettings = new AuthenticationSettings();
+
+        InitializePublicClientApplicationBuilder();
+
+        InitializePublicClientAppForWAMBrokerAsync();
+    }
+
+    [System.Diagnostics.CodeAnalysis.SuppressMessage("Globalization", "CA1305:Specify IFormatProvider", Justification = "Globalization not required as the strings are not being displayed to user and are used in initialization")]

```

Comment:
You should be able to use [invariantculture](https://learn.microsoft.com/en-us/dotnet/api/system.globalization.cultureinfo.invariantculture?view=net-8.0) here instead of this suppress.

Reply:
Neat! I made a change to use this in recent commit. Thanks!